commands/create_keypair.go

```diff
@@ -132,7 +147,8 @@ func verifyCreateKeypairPreconditions(args createKeypairArguments) error {
 }
 
 func createKeyPairRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
-	args := defaultCreateKeypairArguments()
+	args, _ := defaultCreateKeypairArguments()
+
```

Comment:
why here err is not checked?

Reply:
Because this is the second time `defaultCreateKeypairArguments()` is called. If errors occur, that should happen in the first call.

This isn't great design. And deserves rethinking, but I'd rather not do that now, because it affects almost all commands.